pilot_experiment/2_explore_corr_of_genes.ipynb

```diff
@@ -1397,188 +1405,293 @@
+      "text/plain": [
+       "<Figure size 432x432 with 2 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "# Get bins using all data\n",
+    "hist, bins_expression = np.histogram(np.concatenate([mean_real_core_expression,\n",
+    "                                              mean_real_acc_expression]))\n",
+    "\n",
+    "\n",
+    "# Set up the matplotlib figure\n",
+    "fig, axes = plt.subplots(ncols=2, nrows=1, figsize=(6,6))\n",
+    "\n",
+    "# Distribution plot for core genes\n",
+    "sns.distplot(mean_real_core_expression, \n",
+    "             label='core', \n",
+    "             color='red',\n",
+    "             bins=bins_expression,\n",
+    "             kde=False,\n",
+    "             ax=axes[0]\n",
+    "            )\n",
+    "\n",
+    "sns.distplot(mean_real_acc_expression,\n",
+    "             label='accessory',\n",
+    "             color='blue',\n",
+    "             bins=bins_expression,\n",
+    "             kde=False,\n",
+    "             ax=axes[1]\n",
+    "            )\n",
+    "\n",
+    "plt.suptitle('Histogram of gene expression per group')\n",
+    "axes[0].set_title('Core-Core')\n",
+    "axes[1].set_title('Accessory-Accessory')\n",
+    "fig.text(0.5, 0.01, 'Gene expression', ha='center', fontsize=12)\n",
+    "fig.text(0.01, 0.5, 'Count', ha='center', rotation=90, fontsize=12)\n",
+    "plt.tight_layout(pad=0.4, \n",
+    "                 w_pad=0.5,\n",
+    "                 h_pad=1.0,\n",
+    "                 rect=[0, 0.03, 1, 0.95])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**Take away**:\n",
+    "* If high corr(acc, acc) was due to gene absence, then I would’ve expected the average expression of accessory genes to be tightly spread near 0. But instead we see that average gene expression of accessory genes is similar to the core gene expression, even using only PA14 strains\n",
+    "* What can explain this? "
```

Comment:
Yeah, this is interesting. To me, it doesn't look like the expression of the accessory genes is substantially different from the core genes, maybe slightly lower but definitely not close to 0 in most cases.

As an aside, what are the units used to quantify gene expression here? Since this is from ADAGE this is microarray data, right?

Reply:
Very good observation. So this data was using arrays to quantify expression which is why there are no 0 expressed values for those absent genes. This is motivating our next step, which is to use rna-seq instead